pkg/build/builder/docker_test.go

```diff
@@ -0,0 +1,87 @@
+package builder
+
+import (
+	"testing"
+)
+
+func TestReplaceValidCmd(t *testing.T) {
```

Comment:
I can add a test where we'll parse both the original and the new Dockerfile and compare their ASTs. Validation of the Dockerfile though is something different than what this change is supposed to do. @bparees what do you think?